compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -17388,82 +17388,47 @@ TR::Node *lucmpSimplifier(TR::Node *node, TR::Block *block, TR::Simplifier * s)
 
 TR::Node *imaxminSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
    {
+   TR_ASSERT(node->getNumChildren() == 2, "max/min node should only support 2 children");
    simplifyChildren(node, block, s);
 
-   int32_t i = 0;
-   int32_t num_children = node->getNumChildren();
+   TR::Node * firstChild = node->getFirstChild();
+   TR::Node * secondChild = node->getSecondChild();
    int32_t min = 0, max = 0;
    uint32_t umin = 0, umax = 0;
-   bool is_all_const = true;
-   bool has_const = false;
-   int32_t numNonConst = 0;
+   bool is_both_const = firstChild->getOpCode().isLoadConst() && secondChild->getOpCode().isLoadConst();
    bool is_signed  = node->getOpCodeValue() == TR::imax || node->getOpCodeValue() == TR::imin;
    bool max_opcode = node->getOpCodeValue() == TR::imax || node->getOpCodeValue() == TR::iumax;
-   TR::Node * constNode = NULL;
 
-   for (i = 0; i < num_children; i++)
+   if (is_both_const)
```

Comment:
I would vote to stay consistent with the rest of the codebase and rename these variables to camelCase. I know you're keeping the code like it was before but there is no better time to fix this than now. Consistency is king when it comes to readability of code. Similar comment for other areas this PR touches.

Reply:
Fixed in a2fb988